includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```diff
@@ -112,6 +112,12 @@ public function __construct() {
 		$this->secret_key           = ! empty( $main_settings['secret_key'] ) ? $main_settings['secret_key'] : '';
 		$this->statement_descriptor = ! empty( $main_settings['statement_descriptor'] ) ? $main_settings['statement_descriptor'] : '';
 
+		// When feature flags are enabled, title shows the count of enabled payment methods in settings page only.
+		if ( WC_Stripe_Feature_Flags::is_upe_checkout_enabled() && WC_Stripe_Feature_Flags::is_upe_preview_enabled() && isset( $_GET['page'] ) && 'wc-settings' === $_GET['page'] ) {
+			$enabled_payment_methods_count = count( $this->get_upe_enabled_payment_method_ids() );
+			$this->title                   = $enabled_payment_methods_count . ( $enabled_payment_methods_count > 1 ? ' payment methods' : ' payment method' );
```

Comment:
This should probably be wrapped in a translation function (`_n(...)` - https://codex.wordpress.org/I18n_for_WordPress_Developers#Plurals)

Reply:
Thank you. Addressed :tada: